.golangci-new.yml

```diff
@@ -15,6 +15,8 @@ linters-settings:
     check-type-assertions: false
     check-blank: false
     disable-default-exclusions: false
+    exclude-functions:
+      - (*database/sql.Rows).Close
```

Comment:
I'm n;ot sure why we're ignoring that.

Reply:
Because the error should be checked by from https://pkg.go.dev/database/sql#Rows.Err. See https://pkg.go.dev/database/sql#example-Rows